docs/draft-dontula-lsr-yang-dynamic-flooding-02.xml

```diff
@@ -1353,15 +1375,15 @@ module ietf-isis-dynflood {
      </t>
      <figure align="left">
        <artwork align="left"><![CDATA[
-        name: ietf-ospf-df
-        namespace: urn:ietf:params:xml:ns:yang:ietf-dynamic-flooding-ospf
-        prefix: ospf-df
-        reference: RFC XXXX
-
-        name: ietf-isis-dynflood
-        namespace: urn:ietf:params:xml:ns:yang:ietf-isis-dynflood
-        prefix: isis-dynflood
-        reference: RFC XXXX
+       name: ietf-ospf-df
+       namespace: urn:ietf:params:xml:ns:yang:ietf-dynamic-flooding-ospf
+       prefix: ospf-df
+       reference: RFC XXXX
+
+       name: ietf-isis-dynflood
+       namespace: urn:ietf:params:xml:ns:yang:ietf-isis-dynflood
+       prefix: isis-dynflood
+       reference: RFC XXXX
```

Comment:
Can all of this just go?

Reply:
The section just above this says:
"This document registers two YANG modules in the YANG Module Names registry."
If we delete the ietf-isis-dynflood yang module section or delete 
both ietf-isis-dynflood and ietf-dynamic-flooding-ospf section, then that
comment will also need to go. Is that okay?

 =============================

     <t>
       This document registers two YANG modules in the YANG Module Names
       registry <xref target="RFC6020"/>.
     </t>
     <figure align="left">
       <artwork align="left"><![CDATA[
       name: ietf-ospf-df
       namespace: urn:ietf:params:xml:ns:yang:ietf-dynamic-flooding-ospf
       prefix: ospf-df
       reference: RFC XXXX

       name: ietf-isis-dynflood
       namespace: urn:ietf:params:xml:ns:yang:ietf-isis-dynflood
       prefix: isis-dynflood
       reference: RFC XXXX
]]></artwork>
     </figure>

Reply:
Fine by me.